internal/app/server/open_saves_test.go

```diff
@@ -1241,6 +1241,105 @@ func TestOpenSaves_QueryRecords_Offset(t *testing.T) {
 	require.Equal(t, 2, len(resp.StoreKeys))
 }
 
+func TestOpenSaves_GetMultiRecords_InvalidArguments(t *testing.T) {
```

Comment:
I'm trying to replace `assert` with `cmp`.

So, prefer `if !cmp.Equal(want ,got) { t.Errorf(...) }` and `if diff := cmp.Diff(want, got); diff != "" { ... }`

I'll replace the rest of this file eventually (if possible) so leave it if you're on a tight deadline.